Publish ocis release notes with content (#4736)
owncloud/docs: modules/ROOT/pages/ocis_release_notes.adoc

{empty} +

toc::[]

== Changes in 2.0.0.beta1

Version 2.0.0 is a milestone in the development of Infinite Scale as it now comes with all features that have been planned for the upcoming general availability version. Before general availability, there are going to be a number of beta releases as the last development phase before Infinite Scale will be available and recommended for production use. 

You can read more about the Infinite Scale beta phase on https://owncloud.com

- https://owncloud.com/news/infinite-scale-beta-available/[Welcome to the Infinite Scale Beta]
- https://owncloud.com/infinite-scale-beta[Infinite Scale Beta Page]
- https://owncloud.com/news/infinite-scale-beta/[Infinite Scale Beta is coming]

The first beta release of Infinite Scale 2.0.0 (beta1) introduces the `File Search` feature and completely replaces the integrated user management with a lightweight LDAP server that is shipped out-of-the-box (LibreIDM). ownCloud Web introduces a new, feature-rich upload manager based on uppy.io and comes with a couple of design and usability round-offs.

> The most prominent changes in **ownCloud Infinite Scale 2.0.0 beta1** and ownCloud Web 5.5.0 comprise:

Reviewer: ```suggestion
The most prominent changes in **Infinite Scale 2.0.0 beta1** and ownCloud Web 5.5.0 comprise:
```

> * All `Breaking changes` of Infinite Scale 1.20.0 Technology Preview have been fixed. Especially the https://github.com/cs3org/wopiserver[WOPI Server extension] is compatible with Infinite Scale 2.0.0 again.

Reviewer: ```suggestion
* All `breaking changes` of Infinite Scale 1.20.0 Technology Preview have been fixed. Especially the https://github.com/cs3org/wopiserver[WOPI Server extension] is compatible with Infinite Scale 2.0.0 again.
```

* The existing integrated user management has been replaced by LibreIDM which is a lightweight LDAP server being developed by ownCloud and the LibreGraph community (https://github.com/libregraph/idm). LibreIDM comes with a MS Graph-based API and integrates a user interface for user & group management in ownCloud Web. https://github.com/owncloud/ocis/pull/3331[ocis#3331], https://github.com/owncloud/web/issues/6673[web#6673]

* For security reasons, Infinite Scale will not start anymore without specifying certain secrets. A new command `ocis init` has been added to automatically prepare a new Infinite Scale installation by generating secure default secrets. https://github.com/owncloud/ocis/pull/3551[ocis#3551]

* The `Shares` folder has been moved into a dedicated entry point. It is now available in the left sidebar as `Shares` and will gather all incoming and outgoing shares of the user. With this, the transition to a clear separation of personal, shared and project files is complete. https://github.com/owncloud/web/issues/6448[web#6448]

> * Spaces cover images will now be rendered with an appropriate aspect ratio (16:9). https://github.com/owncloud/web/pull/6829[web#6829]

Reviewer: ```suggestion
* Cover images for spaces will now be rendered with an appropriate aspect ratio (16:9). https://github.com/owncloud/web/pull/6829[web#6829]
```

> * When un-sharing a file/folder, a confirmation dialog will now appear https://github.com/owncloud/web/pull/6795[web#6795]

Reviewer: ```suggestion
* When unsharing a file or folder, a confirmation dialog will now appear. https://github.com/owncloud/web/pull/6795[web#6795]
```

* It is now possible to show/hide file extensions in ownCloud Web using the file list settings. https://github.com/owncloud/web/pull/6793[web#6793]

* Infinite Scale now provides dedicated quick links that can be created using the quick action in the file list. Once created, the quick link will stay until it's removed and will be copied when the quick action is used. https://github.com/owncloud/web/pull/6820[web#6820]

* It is now possible to create links with the `Editor` role for single files. https://github.com/owncloud/web/pull/6787[web#6787]

You can also read the full https://github.com/owncloud/ocis/releases/tag/v2.0.0[ownCloud Infinite Scale changelog] and https://github.com/owncloud/web/releases/tag/v5.5.0[ownCloud Web changelog] for further details on what has changed.

=== Known issues

This section will be updated if known issues are discovered.

> * Copy & Move in ownCloud Web does not work outside of personal spaces https://github.com/owncloud/web/issues/6892[web#6892]

Reviewer: ```suggestion
* Copy & Move in ownCloud Web does not work outside of personal spaces. https://github.com/owncloud/web/issues/6892[web#6892]
```

> * Single file user shares have no download action and thumbnail creation does not work for the recipient

Reviewer: ```suggestion
* Single file user shares have no download action and thumbnail creation does not work for the recipient.
```

> * Received shares cannot be renamed https://github.com/owncloud/ocis/issues/3721[ocis#3721]

Reviewer: ```suggestion
* Received shares cannot be renamed. https://github.com/owncloud/ocis/issues/3721[ocis#3721]
```

> * The personal user quota is currently hardcoded https://github.com/owncloud/ocis/issues/3748[ocis#3748]

Reviewer: ```suggestion
* The personal user quota is currently hardcoded. https://github.com/owncloud/ocis/issues/3748[ocis#3748]
```
